accord-core/src/main/java/accord/impl/DomainCommands.java

```diff
@@ -18,12 +18,10 @@
 
 package accord.impl;
 
-import accord.primitives.Timestamp;
-
-public interface CommandTimeseriesHolder
+public interface DomainCommands
 {
+    enum Kind { COORDINATION, RECOVERY }
+
     CommandTimeseries<?> byId();
     CommandTimeseries<?> byExecuteAt();
```

Comment:
We also talked about storing only byId to halve the work we're doing and reduce the number of deletions we need to issue. Just noting it here for the purposes of book keeping.